lua/lspconfig/ltex.lua

```diff
@@ -4,24 +4,13 @@ local util = require 'lspconfig/util'
 configs.ltex = {
   default_config = {
     cmd = { 'ltex-ls' },
-    filetypes = { 'tex', 'bib', 'markdown' },
+    filetypes = { 'plaintex', 'tex', 'bib', 'markdown' },
```

Comment:
The main issue is we don't have filetype plugins for all of the supported languages built-in to vim/neovim, these are just used to populate a [filetype](http://vimdoc.sourceforge.net/htmldoc/autocmd.html#FileType) [autocmd](http://vimdoc.sourceforge.net/htmldoc/autocmd.html). We could add support for `org` and `bibtex`, but it would be up to the users to manually match against the extension to the filetype, or to find a filetype plugin (I don't think they exist for these languages).

Reply:
(`bibtex` is supported by the builtin `filetype.vim`)